frontend/src/components/Layouts/Header/NavigationMenu/NavigationMenu.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styles from './NavigationMenu.module.scss';
+
+const NavigationMenu = ({ items, menuType }) => {
+  const navClassName =
+    menuType === 'Header' ? styles.headerNav : styles.burgerNav;
+  return (
+    <ul className={navClassName}>
+      {items.map((item, index) => (
+        <li key={index}>
```

Comment:
Don't use index, you have label and to unique values, it's better to take one of them